src/Microsoft.Bcl.HashCode/tests/Configurations.props

```diff
@@ -0,0 +1,7 @@
+<Project DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netfx;
```

Comment:
Why only NetFx?  We can run the same test on netcoreapp.

Reply:
for netcoreapp Bcl.Hashcode is only a shim, so the actual implementation is comming from corelib which is why I only added this on netfx. I'm fine with adding it in netcoreapp anyway if you think is valuable.

Reply:
It’s good to make sure our APIs pass the same tests across frameworks.  It should be “free” here.

Reply:
makes sense, will add.